### PR TITLE
create dataContainer-1.0 feature

### PR DESCRIPTION
**Pull request by njr-11**
Initial PR to create the dataContainer-1.0 feature.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
The builds failed because tests projects didn't have built-in awareness of dataContainer-1.0 as an EE 11 feature.
#build
#libby

**Comment by njr-11** [comment]
#build
#libby

File: dev/com.ibm.websphere.appserver.features/visibility/public/data-1.0/resources/l10n/io.openliberty.data-1.0.properties [hunk 1]
```diff
@@ -16,4 +16,8 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=Jakarta Data provides a repository-based data access pattern where the application defines custom data repository interfaces and the container/runtime provides the implementation.
\ No newline at end of file
+description=Jakarta Data is a programming model for data access that\
+ allows you to represent data with simple Java objects called entities\
+ and define operations on data as methods of a repository interface.\
+ A Jakarta Data provider supplies the implementation. This feature\
+ includes a built-in Jakarta Data provider for relational database access.
```
Comment (is273) [inline dev/com.ibm.websphere.appserver.features/visibility/public/data-1.0/resources/l10n/io.openliberty.data-1.0.properties]: Jakarta Data is a programming model for data access.
You can use Jakarta Data to represent data with Java objects called entities and to define operations in data as methods of a repository interface.
A Jakarta Data provider supplies the repository implementation to an application. 
The data-1.0. feature includes a built-in Jakarta Data provider for relational database access.

File: dev/com.ibm.websphere.appserver.features/visibility/public/dataContainer-1.0/resources/l10n/io.openliberty.dataContainer-1.0.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Jakarta Data is a programming model for data access that\
+ allows you to represent data with simple Java objects called entities\
+ and define operations on data as methods of a repository interface.\
+ A Jakarta Data provider supplies the implementation.\
+ This feature only includes the Jakarta Data 1.0 specification interfaces.\
+ This feature does not include a Jakarta Data provider.
```
Comment (is273) [inline dev/com.ibm.websphere.appserver.features/visibility/public/dataContainer-1.0/resources/l10n/io.openliberty.dataContainer-1.0.properties]: Jakarta Data is a programming model for data access. 
You can use Jakarta Data to represent data with Java objects called entities and to define operations in data as methods of a repository interface.
A Jakarta Data provider supplies the repository implementation to an application.
The dataContainer-1.0. feature only includes the Jakarta Data 1.0 specification interfaces.
The dataContainer-1.0. feature does not include a Jakarta Data provider.

**Review by is273 — DISMISSED** [review]
LGTM

**Comment by njr-11** [comment]
#build
#libby

**Comment by njr-11** [comment]
> https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=51fff02e-d82a-4528-9422-d3a4bb3fc5bf has errors/failures.

This build has only 1 test failure. It is a pre-existing issue in an unrelated component's test bucket that doesn't enable Jakarta Data.

**Review by KyleAure — APPROVED** [review]

**Pull request merged**